packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -154,6 +154,7 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
                 className={ css('ms-BasePicker-input', styles.pickerInput) }
                 ref={ this._resolveRef('input') }
                 onFocus={ this.onInputFocus }
+                onBlur={ this.onInputBlur }
```

Comment:
Chatted with Jon offline and got clarification of this design decision. Made some code adjustments.  Will push in a sec.